clusters/build-clusters/01_cluster/assets/build01-9hdwj-worker-us-east-1a-infra_machineautoscaler.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: "build01-9hdwj-worker-us-east-1a-infra"
   namespace: "openshift-machine-api"
 spec:
-  minReplicas: 1
+  minReplicas: 3
```

Comment:
Can we do 2? Prometheus only creates two of these pods.

Reply:
We currently have 2. The alert is still active.
We tried to removed `/wal` in prometheus but it only fixed problem for a day.
I would like to try 3 and keep it only if it resolves the alert.